src/request-proxy.js

```diff
@@ -0,0 +1,52 @@
+const http = require('node:http');
+const https = require('node:https');
+
+const { getConfigValue, isValidUrl, color } = require('./util.js');
+
+const LOG_HEADER = '[Request Proxy]';
+
+function initRequestProxy() {
+    try {
+        const { ProxyAgent } = require('proxy-agent');
+        const proxyEnabled = getConfigValue('requestProxy.enabled', false);
```

Comment:
Please allow overriding the config values on the command-line.

Maybe something like:
```
// ...
    }).option('requestProxyEnabled', {
        type: 'boolean',
        default: null,
        describe: 'Enables request proxy',
    }).option('requestProxyUrl', {
        type: 'string',
        default: null,
        describe: 'Request proxy URL',
    }).option('requestProxyBypasss', {
        type: 'array',
        default: null,
        describe: 'Request proxy bypass',
    }).parseSync();

const proxyEnabled = cliArguments.requestProxyEnabled ?? getConfigValue('requestProxy.enabled', false);
const proxyUrl = cliArguments.requestProxyUrl ?? getConfigValue('requestProxy.url', '');
const proxyBypass = cliArguments.requestProxyBypasss ?? getConfigValue('requestProxy.bypass', []);
```